### PR TITLE
Fix `make download` Command

### PR DESCRIPTION
**Pull request by prikhi**
I tried to run a local server on Debian and `make download` was not working for me, giving me `jq command not found` and `cache: no such file or directory` errors. This PR fixes those issues allowing `make download` to succeed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,22 +67,25 @@ html:
...
-cache/packages-016:
+cache/packages-016: cache/
 	@echo $(call publishedPackages,0.16) > $@
 
 $(BUILD_DIR)/0.16/index.json: cache/packages-016
 	@mkdir -p $(BUILD_DIR)/0.16
 	@$(MAKE) $(shell cat $<)
 	@jq '(input_filename|ltrimstr("cache/packages/")|rtrimstr("/documentation.json")|capture("(?<name>^.+)\/(?<version>\\d+\\.\\d+\\.\\d+$$)")) + {docs: .} | select(.docs[0]["generated-with-elm-version"] | startswith("0.16"))?' $(shell cat $<) | jq -s '.' > $@
 
+cache/:
```
Comment (klaftertief) [inline Makefile]: There is already a rule with a `cache` target.

File: Makefile [hunk 2]
```diff
@@ -67,22 +67,25 @@ html:
 	mkdir -p $(BUILD_DIR) && cp -r src/html/ $(BUILD_DIR)
 
 # TODO: find a nice way to keep this DRY and dynamic
-cache/packages-017:
+cache/packages-017: cache/
```
Comment (klaftertief) [inline Makefile]: This should probably be a prerequisite for the `download` rule, so there is no duplication.

**Comment by klaftertief** [comment]
Thanks.
Can you update the PR to point to the binaries in `bin`, then setting the `PATH` should be removed as well.

Did `modd` and `devd` work and only `jq` not?

**Comment by prikhi** [comment]
`modd` & `devd` worked - although the `modd` config specifies an external folder. I figured those `jq` lines didn't work because they were inside `$(shell ... )` expressions. But my knowledge of makefiles doesn't go too deep.

**Comment by klaftertief** [comment]
Makes sense. And yes, the path in the middle file is still from a previous setup. Will need to fix it.

**Comment by prikhi** [comment]
Should be all fixed up now.

**Comment by klaftertief** [comment]
Thanks

**Pull request merged**